include/sacn/cpp/merge_receiver.h

```diff
@@ -100,14 +100,13 @@ class MergeReceiver
      * packets on the universe.
      *
      * @param[in] handle The merge receiver's handle.
-     * @param[in] universe The universe this merge receiver is monitoring.
      * @param[in] source_addr The network address from which the sACN packet originated.
      * @param[in] header The header data of the sACN packet.
      * @param[in] pdata Pointer to the data buffer. Size of the buffer is indicated by header->slot_count. This buffer
      *                  is owned by the library.
      */
-    virtual void HandleNonDmxData(Handle handle, uint16_t universe, const etcpal::SockAddr& source_addr,
-                                  const SacnHeaderData& header, const uint8_t* pdata) = 0;
+    virtual void HandleNonDmxData(Handle handle, const etcpal::SockAddr& source_addr, const SacnHeaderData& header,
```

Comment:
I think I'd prefer to keep the universe as a parameter.  I agree that it is a bit redundant. but the other two callbacks return it and the application might be using the universe as an index to internal data instead of the handle.

Reply:
https://github.com/ETCLabs/sACN/pull/64/commits/dbbd995afe1527a5cd2c810dc20deeea3f60961b